docs/dasharo-tools-suite/documentation.md

```diff
@@ -56,48 +59,53 @@ To access Dasharo Tools Suite:
 
 ### Bootable USB stick
 
-This section describes how to boot DTS using USB stick.
+This section describes how to boot DTS using a USB stick.
 
 #### Requirements
 
-* USB stick (at least 2GB)
-* Wired network connection
-* [Secure Boot disabled](#disabling-secure-boot)
-* Latest image from [releases](./releases.md) section
+Below are the requirements that must be met to run DTS from a USB device on the
+platform:
+
+* USB stick (at least 2GB),
+* wired network connection,
+* [Secure Boot disabled](#disabling-secure-boot),
+* latest image from [releases](./releases.md) section.
```

Comment:
@TomaszAIR and how about `Bios lock` (https://docs.dasharo.com/unified-test-documentation/dasharo-security/20J-bios-lock-support/). Is that feature should not set as disabled?

Reply:
@sulewskiprzemyslaw No, we can boot DTS without BIOS lock disabled.

Reply:
@TomaszAIR https://github.com/Dasharo/docs/pull/369#discussion_r1111688551